mkdocs.yml

```diff
@@ -42,9 +40,19 @@ nav:
     - Introduction: section-4/intro.md
     - Teleoperated Period: section-4/teleop.md
     - Autonomous Period: section-4/auton.md
-  - 5. Advanced Programming:
+  - 5. Unit Testing:
     - Introduction: section-5/intro.md
-    - Additional Topics: section-5/additional-topics.md
+    - Why Unit Tests?: section-5/why-tests.md
+    - Writing Tests: section-5/writing-tests.md
+    - Continuous Integration: section-5/ci.md
+  - 6. Build Season Process:
+    - Introduction: section-6/intro.md
+    - Project Organization: section-6/organization.md
```

Comment:
I suggest renaming this to something like "Branch Organization". The term "Project Organization" is usually used to refer to organization of the code into files and folders.